microsoft-edge/extensions-chromium/publish/contact-extensions-team.md

```diff
@@ -27,4 +27,4 @@ ms.date: 02/17/2021
 <!-- ====================================================================== -->
 ## Report copyright or trademark infringement
 
```

Comment:
re: merge conflict: [main](https://raw.githubusercontent.com/MicrosoftDocs/edge-developer/main/microsoft-edge/extensions-chromium/publish/contact-extensions-team.md) branch contains:

If you think an item in the Microsoft Edge Add-ons website violates a copyright or trademark, fill out the [Reporting Infringement](https://www.microsoft.com/en-us/concern/dmca) form.  The Microsoft Edge extensions team will review your report and then take the necessary action.